README.md

```diff
@@ -1,6 +1,10 @@
-http://psywerx.net/
+# Psywerx.net
 
-# Dev environment
+[![CircleCI](https://circleci.com/gh/Psywerx/psywerx.svg?style=svg)](https://circleci.com/gh/Psywerx/psywerx)
+[![codecov](https://codecov.io/gh/baloncek2662/psywerx/branch/master/graph/badge.svg)](https://codecov.io/gh/baloncek2662/psywerx)
```

Comment:
Tole je kul za tvoj fork, ampak ce mergam to v psywerx repo mora bit link na psywerx/psywerx in ne baloncek2662/psywerx